src/pages/bills/bills.page.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from "react";
+import {
+  Divider,
+  Grid,
+  CircularProgress,
+  Paper,
+  Box,
+  TableContainer,
+  Table,
+  TableBody,
+  TableHead,
+  TableRow,
+  TableFooter,
+  TablePagination,
+  Button,
+} from "@mui/material";
+import { users } from "data/user";
+import { StyledTableCell, StyledTableRow } from "../../components";
+import { Add } from "@mui/icons-material";
+import { Popup } from "./components/popup.component";
+
+export const Bills = () => {
+  const [open, setOpen] = useState(false);
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+
+  const handleChangePage = (
+    event: React.MouseEvent<HTMLButtonElement> | null,
+    newPage: number
+  ) => {
+    setPage(newPage);
+  };
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const [bills, setBills] = useState([
+    {} as {
```

Comment:
Please use an interface here as well, there should be uniformity in the project.